### PR TITLE
Added Flask Docs for #3

### PR DESCRIPTION
**Pull request by basudev-tyagi**
meow!!  i have written docs  for flask :) please have a look

### CODE REVIEW

File: docs/Flask code example/Flask Documentation.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+The endpoint to submit a response will be in the format:
+```
+https://server.meowform.xyz/form/<your_registered_email>&<your_form_name>
+```
+along with the data from the HTML form created by the user!
+
+If the POST request is made sucessfully the endpoint will return status 200 😁 otherwise it will result in some error 😕
+
+We will make use of the ```requests``` module in Python to work with HTTPS methods.<br><br>
```
Comment (tewarig) [inline docs/Flask code example/Flask Documentation.md]: hey these br tags are causing a problem in build either remove them or close them :)

**Review by tewarig — COMMENTED** [review]
please fix following issues

File: docs/Flask code example/Flask Documentation.md (hunk 1 above)
Comment (basudev-tyagi) [reply docs/Flask code example/Flask Documentation.md]: will do it by today..

Comment (tewarig) [reply docs/Flask code example/Flask Documentation.md]: sure :)

**Comment by basudev-tyagi** [comment]
@tewarig issues resolved! please check.

**Comment by tewarig** [comment]
yup it is fixed now .. thanks :)

**Pull request merged**